gnomad_qc/v4/annotations/insilico_predictors.py

```diff
@@ -203,20 +229,37 @@ def import_pangolin_vcf(vcf_path: str) -> hl.Table:
             force_bgz=True,
             reference_genome="GRCh38",
             skip_invalid_loci=True,
+            header_file=header_file_path,
         ).rows()
         return ht
 
     ht_g = import_pangolin_vcf(v4_genomes)
+    ht_g = ht_g.checkpoint(
+        "gs://gnomad-tmp-4day/pangolin_genomes.ht", overwrite=args.overwrite
+    )
+    ht_g_new = import_pangolin_vcf(v4_genomes_new)
+    ht_g_new = ht_g_new.checkpoint(
+        "gs://gnomad-tmp-4day/pangolin_genomes_new.ht", overwrite=args.overwrite
+    )
     ht_e = import_pangolin_vcf(v4_exomes)
+    ht_e = ht_e.checkpoint(
+        "gs://gnomad-tmp-4day/pangolin_exomes.ht", overwrite=args.overwrite
+    )
+    ht_bugfix = import_pangolin_vcf(v4_bugfix)
+    ht_bugfix = ht_bugfix.checkpoint(
+        "gs://gnomad-tmp-4day/pangolin_genomes_bugfix.ht", overwrite=args.overwrite
+    )
+
+    ht_g_correct = ht_g.anti_join(ht_bugfix)
+    ht_g = ht_g_correct.union(ht_bugfix)
 
     logger.info(
-        "Number of rows in genomes Pangolin HT: %s; "
-        "Number of rows in raw exomes Pangolin HT: %s. ",
-        ht_g.count(),
-        ht_e.count(),
+        f"Number of rows in genomes Pangolin HT: {ht_g.count()};\n Number of rows in"
+        f" raw exomes Pangolin HT: {ht_e.count()};\n Number of rows in Pangolin HT for"
+        f" new variants of genomes: {ht_g_new.count()}.\n"
     )
 
-    ht = ht_g.union(ht_e)
+    ht = ht_g.union(ht_e, ht_g_new)
```

Comment:
Mentioned above but a comment on this ht would be helpful, only need it once though despite me commenting twice.